src/goto-instrument/cover.h

```diff
@@ -22,6 +22,10 @@ class message_handlert;
 class cmdlinet;
 class optionst;
 
+#define OPT_COVER "(cover):"
```

Comment:
Thank you for this refactoring.  Finding an removing unnecessary and redundant repetition is always good.